front/app/containers/Admin/projects/edit/general/index.tsx

```diff
@@ -1,576 +0,0 @@
-      },
-    }));
-  };
-
-  onSubmit = (event: FormEvent<any>) => {
-    event.preventDefault();
-
-    const { projectType } = this.state;
-
-    // if it's a new project of type continuous
-    if (projectType === 'continuous') {
-      eventEmitter.emit('getParticipationContext');
-    } else {
-      this.save();
-    }
-  };
-
-  handleParcticipationContextOnSubmit = (
-    participationContextConfig: IParticipationContextConfig
-  ) => {
-    this.save(participationContextConfig);
-  };
-
-  handleStatusChange = (value: PublicationStatus) => {
-    this.setState(({ projectAttributesDiff }) => ({
-      submitState: 'enabled',
-      publicationStatus: value,
-      projectAttributesDiff: {
-        ...projectAttributesDiff,
-        admin_publication_attributes: {
-          publication_status: value,
-        },
-      },
-    }));
-  };
-
-  handleSlugOnChange = (slug: string) => {
-    this.setState(({ projectAttributesDiff }) => {
-      return {
-        slug,
-        projectAttributesDiff: {
-          ...projectAttributesDiff,
-          slug,
-        },
-      };
-    });
-
-    this.validateProjectSlug(slug);
-  };
-
-  validate = () => {
-    const { formatMessage } = this.props.intl;
-    const { currentTenant, projectAttributesDiff, project } = this.state;
-
-    const { hasErrors, titleError } = validate(
-      currentTenant,
-      projectAttributesDiff,
-      project,
-      formatMessage
-    );
-
-    this.setState({
-      titleError: !isEmpty(titleError) ? titleError : null,
-    });
-
-    return !hasErrors;
-  };
-
-  save = async (
-    participationContextConfig: IParticipationContextConfig | null = null
-  ) => {
-    await save.apply(this, [participationContextConfig]);
-  };
-
-  validateProjectSlug = (slug: string) => {
-    const isSlugValid = validateSlug(slug);
-
-    this.setState({
-      showSlugErrorMessage: !isSlugValid,
-      submitState: isSlugValid ? 'enabled' : 'disabled',
-    });
-  };
-
-  handleFieldUpdate = (fieldUpdates: { [fieldPath: string]: any }) => {
-    this.setState((prevState) => {
-      const newState = { ...prevState };
-      set(newState, 'submitState', 'enabled');
-      for (const fieldPath in fieldUpdates) {
-        set(newState, fieldPath, fieldUpdates[fieldPath]);
-      }
-      return deepMerge(prevState, newState);
-    });
-  };
-
-  render() {
-    const {
-      publicationStatus,
-      projectType,
-      titleError,
-      project,
-      projectHeaderImage,
-      projectImages,
-      projectFiles,
-      processing,
-      projectAttributesDiff,
-      areasOptions,
-      areaType,
-      submitState,
-      apiErrors,
-      slug,
-      showSlugErrorMessage,
-      currentTenant,
-    } = this.state;
-
-    const { authUser } = this.props;
-
-    if (
-      !isNilOrError(authUser) &&
-      (!get(this.props, 'params.projectId') ||
-        (get(this.props, 'params.projectId') && project !== undefined))
-    ) {
```

Comment:
Removed this check, I didn't see why it's needed. The only relevant thing it's checking is that the project is there when we have a projectId, but we have checks in other places for that.